All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/README.md

```diff
@@ -0,0 +1,82 @@
+Web Scraping Complaints from the Financial Ombudsman Service
+=============================
+
+#### *Kenneth Quan*
+<kenneth.quan1@nhs.net>
+
+TODO: Add downgit link
+
+> [!WARNING]  
+> This repository will contain detailed discussions of victims of bank fraud.
+>
+> There may be references to mental health problems, self-harm, abuse and suicide.
+>
+> Samaritans suicide helpline: [116-123](tel:116123)
+>
+> Financial Help: [https://www.citizensadvice.org.uk/](https://www.citizensadvice.org.uk/) <br>
+> Debt Advice: [https://www.stepchange.org/](https://www.stepchange.org/) <br>
+> Do not use search engines. Go directly via the links.
+
+## Intoduction
+This repo is designed as training material on the web scraping, data wrangling (also known as [munging](https://en.wikipedia.org/wiki/Data_wrangling)) and text analysis of open score complaints data from the [Financial Ombudsman Service]( https://www.financial-ombudsman.org.uk/). <br>
+<br>
+The work is NOT endorsed by the Financial Ombudsman Service.
+
+## How to use the code
+The main code is found in `create_output.py`. The script contains the following steps.
+
+### Constructing the URL link
+
+The construction of the URL link is made using the function:
+
+```
+url_link_from_function = fos_web_scrap.getting_URL_with_date_range(start_date, end_date, config['web']['seach_term']) 
+```
+The inputs of the function are the date range when the complaints were made and the search term use to find the relevant complaints. The parameters can be found in the `config.tmol` file.
+
+### Web scraping
+
+The web scraping is perform using the function:
+```
+complete_list = fos_web_scrap.get_complete_desision_list(url_link_from_function)
+```
```

Comment:
```suggestion

```python
complete_list = fos_web_scrap.get_complete_desision_list(url_link_from_function)
```

```
Specify language and leave blank lines around code block

Reply:
note that this is missing a closed code block so don't click 'commit suggestion'- you will need to make this change manually

Reply:
changed